src/main/java/org/jabref/gui/preferences/AppearancePrefsTab.java

```diff
@@ -53,10 +54,13 @@ public AppearancePrefsTab(DialogService dialogService, JabRefPreferences prefs)
         darkTheme = new RadioButton("Dark theme");
         darkTheme.setToggleGroup(themeGroup);
 
-        if (prefs.get(JabRefPreferences.FX_THEME).equals(BASE_CSS))
-            lightTheme.setSelected(true);
-        else if (prefs.get(JabRefPreferences.FX_THEME).equals(DARK_CSS))
-            darkTheme.setSelected(true);
+        String cssFileName = prefs.get(JabRefPreferences.FX_THEME);
+        if (StringUtil.isNotBlank(cssFileName)) {
+            if (cssFileName.equals(BASE_CSS))
```

Comment:
Actually if you change the equals condition, e.g. BASE_CSS.equals(cssFileName) you avoid the npe as well

Reply:
and I think you still miss a default value if the property is not set

Reply:
Is it necessary? because ThemeLoader will set Base.css by default